android/tooling/rib-intellij-plugin/src/main/resources/META-INF/plugin.xml

```diff
@@ -14,9 +14,9 @@
     ]]></description>
 
     <change-notes><![CDATA[
-      <strong>0.1.0</strong><br/>
+      <strong>0.1.2</strong><br/>
```

Comment:
Do not replace the existing entries. Instead add a new entry here.

Reply:
Should I just use 0.1.3? I noticed that the existing version is 0.1.2 but I didn't see any entries on the change notes about that version. Thanks

Reply:
Good point. Use  0.1.3. Probably the 0.1.2 version contained some inconsequential change.